plopfile.js

```diff
@@ -67,3 +65,37 @@ module.exports = function (plop) {
     ],
   });
 };
+
+/**
+ * Extract an exported name from an export statement.
+ *
+ * @example
+ * getNameFromExport('export { default as FooBear } from "./FooBear";');
+ * // => 'FooBear'
+ *
+ * @example
+ * getNameFromExport('export type { FooBat } from "./FooBat";');
+ * // => 'FooBat'
```

Comment:
I wonder if, based on the `IconProps` example, it's better (and easier) to order based on file name? I think the only reason we want the sorting is to maintain some readability, and I don't actually care how the type vs. default export is ordered for a specific file as long as they're grouped

Reply:
That sounds way better. And would better match how import statements should be ordered.

Reply:
Wish the eslint imports plugin could order these for us. Looks like it doesn't